packages/components/nodes/llms/Azure OpenAI/AzureOpenAI.ts

```diff
@@ -89,6 +89,14 @@ class AzureOpenAI_LLMs implements INode {
                     {
                         label: 'gpt-35-turbo',
                         name: 'gpt-35-turbo'
+                    },
```

Comment:
these are already available on `Azure ChatOpenAI` no?

Reply:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/52495491-b424-472b-ab6a-7f5afbfb6def)

Reply:
@dkindlund is gpt4 and gpt3.5 supported in the Completion endpoint of Azure? `Azure OpenAI`in Flowise is using Completion endpoint whereas `Azure ChatOpenAI` is using Chat Completion endpoint.

what is the difference between using gpt-4 in `Azure ChatOpenAI` versus `Azure OpenAI`?

Reply:
Hey @HenryHengZJ , you bring up a good point.  I had originally thought that the `llms/Azure OpenAI/AzureOpenAI.ts` was using the legacy `completions` endpoint while the `chatmodels/AzureChatOpenAI/AzureChatOpenAI.ts` uses the new `chat.completions` endpoint -- is that correct?

Regardless, I was able to successfully get the `llms/Azure OpenAI/AzureOpenAI.ts` node to correctly work using both the `gpt-4` and `gpt-4-32k` models in Azure for testing -- so I figured others who use Azure might want similar functionality.

That said, if you have plans to retire/deprecate the `llms/Azure OpenAI/AzureOpenAI.ts` node, I understand if you don't want to accept this PR.

Reply:
yep you are right `llms/Azure OpenAI/AzureOpenAI.ts` was using the legacy completions endpoint while the `chatmodels/AzureChatOpenAI/AzureChatOpenAI.ts` uses the new chat.completions endpoint.

I'm just wondering if gpt-4 and others are supported on the completions endpoint, trying to avoid duplication. But since you have tested and it works, thats good!